src/index.html

```diff
@@ -22,6 +22,349 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <img
+        class="imgblock"
+        src="https://s3-alpha-sig.figma.com/img/833f/9aab/47d660efd384318772b6b8040db831aa?Expires=1730073600&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=GFjFD6-6KNqFBkswmu-BuTPJnotQ8-MD~XFWAHKRTFzBEQqKLe-4Cca9JX55r9scHqqYuwSS3TsIyJCh2iERYUgPUe5fszbMK7Tg0AgZZdstjYggVmO~lCJAm5RABG4y~Pd9KlhMIhkeNGbSjrNGNjL~pLc8xnAb1HFOb19z7kl6MngmQbQxT-1vuKf03PUIUzsEhIR8aicMa2KjYgomPp2TDMN9G7n8WqTgmW1h5Qu-jctRoMRYuvHf4xX9pw5TT2~LSzqKDtR0mjg38R~TXmLshaA9MgAoE7RyK873WOssIDG5MfRQ236zHRF9tZrkvuRgIxDAGL6O9x7hCoKsAg__"
+        alt="blue_logo_40x40"
+      />
+      <nav>
+        <ul class="navItems">
+          <li class="navItems__block">
+            <a
+              href="#"
+              class="is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="navItems__block">
+            <a href="#">SAMSUNG</a>
+          </li>
+          <li class="navItems__block">
+            <a href="#">SMARTPHONES</a>
+          </li>
+          <li class="navItems__block">
+            <a
+              href="#"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="navItems__block">
+            <a href="#">GADGETS</a>
+          </li>
+          <li class="navItems__block">
+            <a href="#">TABLETS</a>
+          </li>
+          <li class="navItems__block">
+            <a href="#">PHOTO</a>
+          </li>
+          <li class="navItems__block">
+            <a href="#">VIDEO</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__imgBlock">
+          <img
+            src="./images/imac.jpeg"
+            alt="iMac"
+          />
+        </div>
+        <div class="card__title">
+          <span>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</span>
+        </div>
+        <div class="card__productCode">
+          <span>Product code: 195434</span>
+        </div>
+        <div class="card__rate">
+          <div class="stars-4">
+            <div></div>
+            <div></div>
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
+          <div>Reviews: 5</div>
+        </div>
+        <div class="priceBlock">
+          <span class="priceBlock__label">Price:</span>
+          <span class="priceBlock__price">$2,199</span>
+        </div>
+        <a
+          class="card__button"
+          data-qa="card-hover"
```

Comment:
The 'data-qa="card-hover"' attribute is correctly added to the 'Buy' link inside the first card, as specified in the task.